### ISSUE TITLE
Request: method to run tests using system installed seqan-apps and Seqan headers

### ISSUE THREAD

**Issue by mr-c** [seqan/seqan#2283]
Debian can use this to check the functionality without recompiling the packages. For example, this can catch regressions from any of the libraries that are dynamically linked in.

When we run these tests we have the entire source tree available. Ideally something like the following would work:

```
mkdir debci
cd debci 
cmake ../ 
make test_installed_binaries_and_headers
```

Does that seem feasible? I'm not clever with CMAKE, so I don't know how to go about this myself.

**Comment by rrahn** [comment]
Please correct me if I am wrong, but you want the `test_installed_binaries_and_headers` to run some tests using the preinstalled headers/binaries (from the debian package manager) without compiling the sources? As if it were the user perspective that is using the apps/lib?

**Comment by mr-c** [comment]
@rrahn Correct. Binaries should be located from the system path and the seqan headers from `/usr/include/seqan`

**Comment by rrahn** [comment]
@mr-c that is actually something I had in mind quite some time now, in order to do continuous deployment/delivery, including a self-testing procedure for the packaged binaries. 
Working one this would be a milestone for the SeqAn3 life cycle were we will separate the apps from the library as external contributions (though still in-house apps). 
But it is likely, that we don't add this for SeqAn2, since the deviation in the infrastructure would unnecessarily complicate things. Anyway, we only expect some minor patch level releases for SeqAn2, which are not allowed to break anything to SeqAn 2.4.

**Comment by mr-c** [comment]
@rrahn We can add reasonable sized patches as part of Debian packaging. So if there is a simple thing that could be done with 2.4.x then we would gladly accept and use it. Should be simple path changes, yes?

Otherwise we look forward to 3.0!

**Comment by rrahn** [comment]
Mhmm, it should be working already with SeqAn2.
I will have a look at it.

**Comment by mr-c** [comment]
Some progress:

```
mkdir build
cd build
cmake ..
rmdir bin
ln -s /usr/lib/seqan/bin bin
ctest --tests-regex 'app_.*' --parallel $(nproc)
```

All tests pass, except for `fiona_illumina` which is not installed

**Comment by marehr** [comment]
Great!
